### DISCUSSION TITLE
Change position of the separator if left pane is selected by only two split panes

### DISCUSSION THREAD

**Discussion by ilias777** [nvim-zh/colorful-winsep.nvim#70] [Q&A] [answered]
Hi. I have a question about changing the position of the separator if the focus is on the left side of the split pane.

To understand: At the moment, if only two panes are active and I am on the left side of the pane, the separator is on the bottom position. And if I am on the right side, the separator is on the top position.

In my logical way, if I have only two split panes and I am on the left side, it's more clear for me that the separator is on top position and if I am in the right split pane the separator is in the bottom position.

It's possible to switch the positions?

<img width="1462" alt="screenshot" src="https://github.com/nvim-zh/colorful-winsep.nvim/assets/39212564/528a75f2-05db-452d-b4e3-6fd91127d952">

Same with horizontal splits:

<img width="1510" alt="screenshot2" src="https://github.com/nvim-zh/colorful-winsep.nvim/assets/39212564/3dea8846-08d5-4080-82b2-b86759d2840b">

Thank you for any help.

**Comment by denstiny** [comment]
no,oh no

**Reply by ilias777** [reply]
No?? Really?
But see please these two screenshots here &rarr; #50 
In the first screenshot is the focus on the left split and the separator on the top. And in the second screenshot is the focus on the right split and the separator on the bottom. Is this behavior possible?

**Reply by denstiny** [reply, marked as answer]
Ok! sir ,now you can update plugin,set only_line_seq = false

**Reply by ilias777** [reply]
You are amazing! Thank you very much 😊

**Reply by denstiny** [reply]
> You are amazing! Thank you very much 😊

Thank you for your support.

**Reply by denstiny** [reply]
By the way, your line spacing is amazing.

**Reply by ilias777** [reply]
Thank you 🙂

**Comment by denstiny** [comment]
If you have been reviewing the code, it seems to be hardcoded, so I may need to make some adjustments.

**Reply by xiaoliyooo** [reply]
Idol, why does this problem still occur even after I set only_line_seq = false
偶像，为什么我设置了only_line_seq = false 还是会与这个问题

<img width="1438" height="823" alt="image" src="https://github.com/user-attachments/assets/dd9c812d-e201-4f09-b2b5-bf4d0bce1ff6" />

**Reply by denstiny** [reply]
> Idol, why does this problem still occur even after I set only_line_seq = false 偶像，为什么我设置了only_line_seq = false 还是会与这个问题
> 
> <img alt="image" width="1438" height="823" src="https://private-user-images.githubusercontent.com/73976311/471187220-dd9c812d-e201-4f09-b2b5-bf4d0bce1ff6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTM4NDI0OTcsIm5iZiI6MTc1Mzg0MjE5NywicGF0aCI6Ii83Mzk3NjMxMS80NzExODcyMjAtZGQ5YzgxMmQtZTIwMS00ZjA5LWIyYjUtYmY0ZDBiY2UxZmY2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA3MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNzMwVDAyMjMxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU1N2FkNGRhZjlkOGQ1MWQzMTZlMzE0ZjM0YjQxNDc4ZGQzOTQ1MzA1OWM2NTljYWY1YjJiYTU5Mzg2YjgxMTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.mTxz1GBlloM5NNOxv0i_nVasArLnsXPlLf1Od2HURQo">

你重新开个问答吧，然后发送你的基础配置我看下
